src/main/java/org/vietabroader/view/OneColumn.java

```diff
@@ -0,0 +1,46 @@
+package org.vietabroader.view;
+
+import org.vietabroader.view.verifier.ColumnVerifier;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class OneColumn extends JPanel {
+    private JTextField textField;
+    private String label;
```

Comment:
This `label` has never been set. That's why the log says `null C` while it is supposed to print the character of all columns.